src/main/java/rx/swing/sources/WindowEventSource.java

```diff
@@ -0,0 +1,88 @@
+/**
+ * Copyright 2015 Pedro Larroy
```

Comment:
Yes, we've been keeping everything under Netflix so far. If we want to change that everywhere, it will be a bigger discussion. :smiley: